README.md

```diff
@@ -3,7 +3,7 @@
 ---
 
 OpenBalena is a platform to deploy and manage connected devices. Devices run
-[balenaOS][balena-os-website], a host operating system designed for running
+[balenaOS][balena-os], a host operating system designed for running
```

Comment:
No reason for this change -- `balena-os` links to the repo but given this is a high level overview it makes sense to link to the marketing site, which `balena-os-website` does.